client/src/components/restaurant/restaurant-item.js

```diff
@@ -36,14 +37,25 @@ class Restaurant extends Component {
         this.setState({
           restaurant: response.data,
           isFetching: true,
+          // eslint-disable-next-line eqeqeq
+          isOwner: response.data.personId == localStorage.getItem('userID'),
         });
       });
   }
 
   render() {
-    const { isFetching, restaurant } = this.state;
+    const { isFetching, restaurant, isOwner } = this.state;
     const { match: { params: { id } }, isAuthenticated } = this.props;
 
+    let isOwnerText;
+    if (isOwner) {
+      isOwnerText = (
+        <Container>
+          <h4>(You are the owner of this restaurant)</h4>
```

Comment:
Why you use brackets?

Reply:
They will be shown on rest page. Without them the taxt may look like title or smth else (part of our UI design)...
This text must be like a note. Insted brackets we can put "!" or "*" in front of text, to show thats it's just a note for owner,

Reply:
You can use specific style, other background color, or something else